docs/content/en/docs/reference/clusterspec/vsphere.md

```diff
@@ -177,37 +177,37 @@ certificate is being used, `insecure` must be set to true.
 ## VSphereMachineConfig Fields
 
 ### memoryMiB (optional)
-Size of RAM on virtual machines (Default: 8192)
+Size of RAM on virtual machines. (Default: 8192)
 
 ### numCPUs (optional)
-Number of CPUs on virtual machines (Default: 2)
+Number of CPUs on virtual machines. (Default: 2)
 
 ### osFamily (optional)
-Operating System on virtual machines. Permitted values: ubuntu, bottlerocket (Default: bottlerocket)
+Operating System on virtual machines. Permitted values: ubuntu, bottlerocket. (Default: bottlerocket)
 
 ### diskGiB (optional)
-Size of disk on virtual machines if snapshots aren't included (Default: 25)
+Size of disk on virtual machines if snapshots aren't included. (Default: 25)
 
 ### users (optional)
-The users you want to configure to access your virtual machines. Only one is permitted at this time
+The users you want to configure to access your virtual machines. Only one is permitted at this time.
 
 ### users[0].name (optional)
-The name of the user you want to configure to access your virtual machines through ssh.
+The name of the user you want to configure to access your virtual machines through SSH.
 
 The default is `ec2-user` if `osFamily=bottlrocket` and `capv` if `osFamily=ubuntu`
 
 ### users[0].sshAuthorizedKeys (optional)
-The SSH public keys you want to configure to access your virtual machines through ssh (as described below). Only 1 is supported at this time.
+The SSH public keys you want to configure to access your virtual machines through SSH (as described below). Only 1 is supported at this time.
 
 ### users[0].sshAuthorizedKeys[0] (optional)
-This is the SSH public key that will be placed in `authorized_keys` on all EKS Anywhere cluster VMs so you can ssh into
+This is the SSH public key that will be placed in `authorized_keys` on all EKS Anywhere cluster VMs, so you can SSH into
```

Comment:
SSH is the technology, so I think the command form `ssh` is a better alternative